apache-solr/README.md

```diff
@@ -0,0 +1,140 @@
+# Apache Solr Mixin
+
+The Apache Solr mixin is a set of configurable Grafana dashboards and alerts.
+
+The Apache Solr mixin contains the following dashboards:
+
+- Apache Solr cluster overview
+- Apache Solr query performance overview
+- Apache Solr resource monitoring overview
+- Apache Solr logs overview
+
+and the following alerts:
+
+- ApacheSolrZookeeperChangeInEnsembleSize
+- ApacheSolrHighCPUUsageCritical
+- ApacheSolrHighCPUUsageWarning
+- ApacheSolrHighHeapMemoryUsageCritical
+- ApacheSolrHighHeapMemoryUsageWarning
+- ApacheSolrLowCacheHitRatio
+- ApacheSolrHighCoreErrors
+- ApacheSolrHighDocumentIndexing
+
+## Apache Solr Cluster Overview
+
+The Apache Solr cluster overview dashboard provides details on cluster, shard, replica and Zookeeper health as well as top core and error metrics.
+
+![Apache Solr Cluster Overview Dashboard 1](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-cluster-1.png)
+![Apache Solr Cluster Overview Dashboard 2](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-cluster-2.png)
+
+## Apache Solr Query Performance Overview
+
+The Apache Solr query performance overview dashboard provides details on various query load and latency, update handlers, cache, timeout and error metrics.
+
+![Apache Solr Query Performance Overview Dashboard 1](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-query-performance-1.png)
+![Apache Solr Query Performance Overview Dashboard 2](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-query-performance-2.png)
+
+## Apache Solr Resource Monitoring Overview
+
+The Apache Solr resource monitoring overview dashboard provides details on connections, threads, core FS usage, as well as JVM and Jetty metrics.
+
+![Apache Solr Resource Monitoring Overview Dashboard 1](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-resource-monitoring-1.png)
+![Apache Solr Resource Monitoring Overview Dashboard 2](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-resource-monitoring-2.png)
```

Comment:
Note this may be increased in terms of amount of screenshots if we change the number of columns.

Reply:
Updated screenshots